estimagic/estimation/estimate.py

```diff
@@ -0,0 +1,83 @@
+import numpy as np
+
+from estimagic.config import DEFAULT_DATABASE_NAME
+from estimagic.decorators import aggregate_criterion_output
+from estimagic.optimization.optimize import maximize
+from estimagic.optimization.process_arguments import process_optimization_arguments
+
+
+def estimate_likelihood(
+    criterion,
+    params,
+    algorithm,
+    criterion_kwargs=None,
+    constraints=None,
+    general_options=None,
+    algo_options=None,
+    gradient_options=None,
+    logging=DEFAULT_DATABASE_NAME,
+    log_options=None,
+    dashboard=False,
+    db_options=None,
+):
+    """Estimate parameters via likelihood.
```

Comment:
I think we should call `estimate_likelihood` `maximize_log_likelihood` instead. That makes it immediately obvious that we are maximizing. If we decide to keep the name as is, I would clarify "via maximizing the likelihood" in the first line of the docstring.

Reply:
Done